robottelo/ui/factory.py

```diff
@@ -361,6 +362,42 @@ def make_hostgroup(session, org=None, loc=None, force_context=True, **kwargs):
     Hostgroup(session.browser).create(**create_args)
 
 
+def make_host(session, org=None, loc=None, force_context=True, **kwargs):
+    """Creates a host"""
+
+    create_args = {
+        u'arch': None,
+        u'cpus': '1',
+        u'cv': None,
+        u'custom_ptable': None,
+        u'domain': None,
+        u'env': None,
+        u'host_group': None,
+        u'ip_addr': None,
+        u'lifecycle_env': None,
+        u'loc': None,
+        u'mac': None,
+        u'media': None,
+        u'memory': '768 MB',
+        u'name': None,
+        u'org': None,
```

Comment:
On line 365, you've added this:

```python
def make_host(session, org=None, loc=None, force_context=True, **kwargs):
```

There are `org` and `loc` arguments, and they're passed to `core_factory` below:

```python
core_factory(..., org=org, loc=loc, ...)
```

We also list `org` and `loc` in this `create_args` dict. Is that a conflict?